### PR TITLE
Add lab check periodic job config

### PR DESCRIPTION
**Pull request by Yikun**

### CODE REVIEW

File: playbooks/check-nodepool-periodic.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        name: openlabcmd
+        executable: pip3
+
+- name: Teardown labcheck crontab tasks for nodepool and zuul slave
+  hosts: nodepool-launcher-slave:zuul-scheduler-slave
+  become: yes
+  tasks:
+    - name: Teardown crontab tasks if exists(when slave master switched)
+      cron:
+        name: "{{ item }}"
```
Comment (huangtianhua) [inline playbooks/check-nodepool-periodic.yaml]: If there is only one cron entity, we can put the name here directly, but not blocker.

File: playbooks/check-nodepool-periodic.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        state: absent
+      with_items:
+        - "labcheck nodepool check report script"
+
+- name: Cron task of labcheck for nodepool
+  hosts: nodepool-launcher:!*-slave
```
Comment (huangtianhua) [inline playbooks/check-nodepool-periodic.yaml]: I think this play can be merged with first one play in line13, they are all execute in hosts(nodepool-launcher:!*-slave).

File: playbooks/check-nodepool-periodic.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          fi
+        executable: /bin/bash
+
+    - name: Install openlabcmd
+      pip:
+        name: openlabcmd
+        executable: pip3
+
+- name: Teardown labcheck crontab tasks for nodepool and zuul slave
+  hosts: nodepool-launcher-slave:zuul-scheduler-slave
```
Comment (huangtianhua) [inline playbooks/check-nodepool-periodic.yaml]: Here I think you only need to teardown labcheck crontab in nodepool-launcher-slave.

File: labcheck/lab_check_report.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+cd ~/labcheck/openlab/
+issue_header="[Labcheck] "`date +%Y%m%d%H%M`" OpenLab Environment Check Failed"
+issue_content="Check report as below:\n\`\`\`"
+echo -e $issue_header"\n\n"$issue_content > ~/labcheck/labcheck.issue
+
+openlab check --nocolor >> ~/labcheck/labcheck.issue
+retval=$?
+
+echo -e "\`\`\`">> test.issue;
```
Comment (huangtianhua) [inline labcheck/lab_check_report.sh]: No need semicolon here.

Comment (Yikun) [reply labcheck/lab_check_report.sh]: done

File: playbooks/check-nodepool-periodic.yaml (hunk 3 above)
Comment (Yikun) [reply playbooks/check-nodepool-periodic.yaml]: done

File: playbooks/check-nodepool-periodic.yaml (hunk 1 above)
Comment (Yikun) [reply playbooks/check-nodepool-periodic.yaml]: done

File: playbooks/check-nodepool-periodic.yaml (hunk 2 above)
Comment (Yikun) [reply playbooks/check-nodepool-periodic.yaml]: done

**Comment by huangtianhua** [comment]
Thanks! LGTM

**Review by huangtianhua — APPROVED** [review]

**Comment by Yikun** [comment]
After openlabcmd 1.1.1 release, we can merge this. Current, it's 1.1.0

[1] https://pypi.org/project/openlabcmd/

cc @wangxiyuan @bzhaoopenstack

**Review by kiwik — APPROVED** [review]
LGTM, thank you

**Comment by Yikun** [comment]
openlabcmd has been released as 1.2.0, now we can merge it!
[1] https://pypi.org/project/openlabcmd/

**Pull request merged**